src/ripple/app/misc/SHAMapStore.h

```diff
@@ -69,39 +68,36 @@ class SHAMapStore
-     * historical ledgers over the peer to peer network.
-     * If online_delete is not enabled, then the minimum value to
-     * keep online is the minimum ledger that has been persisted already.
-     *
-     * With online_delete,this value is governed by the
-     * following circumstances:
-     *
-     * Upon process startup:
-     *
-     * With advisory_delete enabled and online_delete having been executed
-     * previously:
-     * Upon process startup, this value is set to one past the value
-     * that is allowed to be deleted. In other words, anything greater
-     * than what can be deleted should be acquired from the network and
-     * then retained. If nothing has yet been deleted, then do not
-     * rely upon the value of what is allowed to be deleted because it
-     * could cause fetching of more history than configured by
-     * ledger_history.
-     *
-     * Without advisory_delete or if online_delete has never executed:
-     * Upon process startup, this value is set to the earliest ledger
-     * that has been persisted in SQLite.
-     *
-     * Each time online_delete executes:
-     *
-     * Just prior to clearing SQL databases of historical ledgers, move
-     * the value forward to one past the greatest ledger being deleted.
-     * This minimizes fetching of ledgers that are in the process of being
-     * deleted.
-     */
-    virtual boost::optional<LedgerIndex> minimumOnline() const = 0;
+    /** The minimum ledger to try and maintain in our database.
+
+        This defines the lower bound for attempting to acquire historical
+        ledgers over the peer to peer network.
+
+        With online_delete,this value is governed by the following
+        circumstances:
+
+        - With advisory_delete enabled and online_delete having been executed
+          previously, this value is set to one past the value that is allowed to
+          be deleted. In other words, anything greater than what can be deleted
+          should be acquired from the network and then retained. If nothing has
+          yet been deleted, then do not rely upon the value of what is allowed
+          to be deleted because it could cause fetching of more history than
+          configured by ledger_history.
+
+        - Without advisory_delete or if online_delete has never executed, this
+          value is set to the earliest ledger that has been persisted in the
+          SQL database.
+
+        Each time online_delete executes, just prior to clearing SQL databases
+        of historical ledgers, move the value forward to one past the greatest
+        ledger being deleted. This minimizes fetching of ledgers that are in
+        the process of being deleted.
+
+        @return If online deletion is enabled and if it has been executed,
+		the minimum ledger to keep available; an unseated
+                optional otherwise.
+    */
+        virtual boost::optional<LedgerIndex> minimumOnline() const = 0;
```

Comment:
Extra spaces here. This can be probably resolved with `clang-format` better than you manually editing the file. We can do this during merge.